drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -570,11 +565,12 @@ static int lis2dh_pm_action(const struct device *dev,
  * Instantiation macros used when a device is on an I2C bus.
  */
 
+#include <zephyr/drivers/i2c.h>
```

Comment:
nit: includes should be at the top

Reply:
Apologize for that... Added it to get autocompletion of macros in vscode and forgot to remove is. 🤦